crd-docs/crd/kiali.io_kialis.yaml

```
@@ -745,9 +745,18 @@ spec:
                       envoy_admin_local_port:
                         description: "The port which kiali will open to fetch envoy config data information."
                         type: integer
-                      gateway_api_class_name:
-                        description: "The K8s Gateway API GatewayClass's Name used in Istio. If empty, the default value 'istio' is used."
-                        type: string
+                      gateway_api_classes:
+                        description: "The K8s Gateway API GatewayClass's implementation name and class name used in Istio. If empty, the default value 'Istio/istio' is used."
```

Since this is now an array, we should re-word this to make that clear.
```suggestion
                        description: "A list declaring all the Gateways used in Istio. If left empty or undefined, the default is a single list item whose name is `Istio` and class_name is `istio`."
```